### DISCUSSION TITLE
Section "Quote" disappears after compiling

### DISCUSSION THREAD

**Discussion by Fredholme** [joaomlourenco/novathesis#156] [Q&A] [answered]
Hi professor,

I am currently using version 5.1.3 of novathesis template and I noticed that when you compile the document for the first time, the section "quote" vanishes. I tried substituting "filename" to "quote" but it remained the same.

Best regards.

**Comment by joaomlourenco** [comment, marked as answer]
Hello. 
Did you compile is "as downloaded"? Or did you change them document type or something else?

**Reply by Fredholme** [reply]
Hi.
I compiled it immediately after I downloaded it.

**Reply by joaomlourenco** [reply]
Weired.  Can you please doble check.  It works for me.

**Reply by Fredholme** [reply]
I decided to download the template again and repeat the procedure. The result was the same. The "quote" section appears in the bookmarks of the template, however, when you click on it you get the "abstract"... somehow it is being omitted.

**Reply by joaomlourenco** [reply]
Got it.  Just open 'template.tex' and replace
```latex
\ntaddfile{quote}{filename}
```
with
```latex
\ntaddfile{quote}{quote}
```

**Reply by Fredholme** [reply]
Hi professor,

As I said previously, that change has been done before and nothing has changed. Recently, I talked with a colleague that has the same issue, but that is not all... not only his "quote" section does not show, his "bibliography" also vanishes after 1st compiling (i.e., no changes after download). I, however, do not have this issue about the bibliography.

**Reply by joaomlourenco** [reply]
I did add 
```latex
\ntaddfile{quote}{quote}
```
and it worked for me, and bibliography works all right.   Is your friend trying to use "bibtex"?  He/she must use "biber"!

Remember to check the errors in the "template.log" file to find out what is really happening.

**Reply by Fredholme** [reply]
I'm sorry professor, but neither solution (to the quote and bibliography sections) is working for us in the most recent version of the template.

**Reply by joaomlourenco** [reply]
Both the “quote” and “bibliography” sections **are working** in the latest version of the template. *I just tried it now 2021-03-04@19:32!*

Please check your “template.log” file and search for both "Error" and "error" and find out what your problem is!

**Reply by Fredholme** [reply]
These are the two lines where the word "error" appears on my template.log:
1) Package: infwarerr 2019/12/03 v1.5 Providing info/warning/error messages (HO)
2) !pdfTeX error: pdflatex (file zxxrow7z): Font zxxrow7z at 498 not found
 ==> Fatal error occurred, no output PDF file produced!
I don't understand the second statement because my template runs and creates the pdf.

**Reply by joaomlourenco** [reply]
Look for "Error" as well.  You must have some other serious error.

**Reply by Fredholme** [reply]
I searched for "Error" as well, but it did not find any.

**Reply by Fredholme** [reply]
Hi professor,

I have recently switched from LaTeXstudio to Overleaf and everything works fine.

**Reply by joaomlourenco** [reply]
👍

**Comment by joaomlourenco** [comment]
Oh! You are using 5v.1.3!
Please update to v6.x.  It will work as expected…
